string name
string jointName
float64 angle
